### ISSUE TITLE
Arrow icons overlap skill cards

### ISSUE THREAD

**Issue by arundhati24** [fossasia/susi_skill_cms#873]
**Actual Behaviour**

Arrow icons overlap skill cards.

**Expected Behaviour**

Either the arrow icons should not overlap the skill cards and if they do, it should be done equally on both sides.

**Screenshot of the issue**

<img width="1438" alt="screen shot 2018-06-27 at 10 48 39 am" src="https://user-images.githubusercontent.com/30979369/41954198-b7c05a9a-79f7-11e8-8357-1da76e3756f1.png">

**Would you like to work on the issue?**

No, anybody is free to take this up.

**Comment by Akshat-Jain** [comment]
I'll work on this.

**Comment by anshumanv** [comment]
@arundhati24 thanks for reporting this but in your screenshot top skills aren't loaded so it looks that way, how it actually is - 
![image](https://user-images.githubusercontent.com/21009455/41983424-ae8e8786-7a4b-11e8-8605-539dca13f2f3.png)
And thus the design is in accordance with alexa
Ref(Alexa) - 
![image](https://user-images.githubusercontent.com/21009455/41983434-b80fdc6a-7a4b-11e8-9792-f85f986ee5a7.png)

**Comment by Akshat-Jain** [comment]
@anshumanv We can show the arrow icons only when the top rated skills have been fetched. The arrow icons seem misaligned till the top rated skills have been fetched otherwise.

**Comment by anshumanv** [comment]
@Akshat-Jain fixed in https://github.com/fossasia/susi_skill_cms/pull/854

**Comment by anshumanv** [comment]
Yes, there have been issues lately while fetching server data thus it looks that way sometimes. No worries.
